src/Components/test/E2ETest/Tests/StandaloneAppTest.cs

```diff
@@ -126,10 +126,12 @@ private void WaitUntilLoaded()
         Browser.NotEqual("Loading...", () => app.Text);
     }
 
+#if !PLAYWRIGHT_ADAPTER
     public void Dispose()
     {
         // Make the tests run faster by navigating back to the home page when we are done
         // If we don't, then the next test will reload the whole page before it starts
         Browser.Exists(By.LinkText("Home")).Click();
     }
+#endif
```

Comment:
It's probably possible to make this work without being `#if`-ed out. I only disabled it because of an execution ordering issue (it was tearing down the Playwright browser first, then calling this, which threw because it could no longer talk to the browser).